osf_tests/test_collection.py

```diff
@@ -4,6 +4,7 @@
 
 from framework.auth import Auth
 
+from osf.models import Collection
```

Comment:
Will this file also have general collections tests, otherwise might consider renaming to test_bookmark_collection

Reply:
It is where `Collection`-related model tests should go, yes